meinberlin/assets/scss/utility.scss

```diff
@@ -154,6 +154,10 @@
     overflow-wrap: break-word;
 }
 
+.u-overflow-visible {
+    overflow: visible;
+}
+
```

Comment:
I know this is a problem somewhere else but can't remember where but thought was worth a utility till I can remember